src/OXRS_API.cpp

```diff
@@ -213,7 +213,13 @@ void _getMqtt(Request &req, Response &res)
 void _postMqtt(Request &req, Response &res)
 {
   DynamicJsonDocument json(2048);
-  deserializeJson(json, req);
+
+  DeserializationError error = deserializeJson(json, req);
+  if (error) 
```

Comment:
could be shorter I think ;-) 

if (deserializeJson(json, req))
{

Reply:
Yep - but I think that is a little confusing to someone reading the code - it looks like it is failing if `deserializeJson` is successful...